activestorage/test/models/attached/many_test.rb

```diff
@@ -593,4 +593,16 @@ def highlights
       User.remove_method :highlights
     end
   end
+
+  test "loading associations after attaching" do
+    funky = create_blob(filename: "funky.jpg")
+    town = create_blob(filename: "town.jpg")
+
+    @user.highlights.attach(funky)
+    @user.reload
+    @user.highlights.attach(town)
+
+    assert_not_empty @user.highlights_attachments
+    assert_equal @user.highlights_blobs.count, 2
```

Comment:
Can you please add these assertions to the first test (named “attaching existing blobs to an existing record”) and remove this one?